_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,318 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that for any text content that auto-updates in a 10 minute time span, there are instruments to pause, stop, or hide it or to control its update frequency. The arbitrary 10 minute time span, selected so that testing this rule would not be impractical, is not included in WCAG. Content that updates less frequently may fail success criteria 2.2.2 without failing this rule.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML element][] that has a [visible][] [text node][] as a [descendant][] in the [flat tree][] if:
+
+- **changed:** the `innerText` property of the [element][html element] changes multiple times within a 10 minute time span where there is no [user interaction][]; and
```

Comment:
Now that we've worked out (or will soon have) a definition of "changes in content" that includes a definition of "visible changes", I'm wondering whether we should upgrade this rule to use it (the "visible changes" part only, of course) 😕 

WDYT?

Reply:
@Jym77 that would imply a big change in the applicability (because "changes in content" applies to the web page, not specific elements). And, at least, it would make it difficult to specify that it is always the same content in the page that is being changed.